Assets/MRTK/Core/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -229,15 +234,7 @@ private static bool UpdateVSProj(IBuildInfo buildInfo)
         {
             // For ARM64 builds we need to add ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch
             // to vcxproj file in order to ensure that the build passes
-
-            string projectName = PlayerSettings.productName;
-            string projectFilePath = Path.Combine(Path.GetFullPath(buildInfo.OutputDirectory), projectName, $"{projectName}.vcsproj");
-
-            if (!File.Exists(projectFilePath))
-            {
-                Debug.LogError($"Cannot find project file: {projectFilePath}");
-                return false;
-            }
+            string projectFilePath = GetProjectFilePath(buildInfo);
```

Comment:
Should this check for null and return false if so, like the previous failure case did?

Reply:
```suggestion
            string projectFilePath = GetProjectFilePath(buildInfo);
            if (projectFilePath == null)
            {
                return false;
            }
```